src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return String.valueOf(value);
```

Comment:
```suggestion
            return "0";
```
it's always will be "0", so i don't see reason not to return "0" right away

Reply:
Да так лучше :D